src/v4.rs

```diff
@@ -1,5 +1,5 @@
 use crate::prelude::*;
-use rand;
+use getrandom::getrandom;
```

Comment:
`uuid` uses the import module only policy. As `getrandom` is a crate and we are using `2018` edition, we don't need this line at all now.

```suggestion
```

Reply:
Done.